confounds/metrics.py

```diff
@@ -6,3 +6,61 @@
 degree of harmonization achieved (e.g. reduction in variance of means/medians)
 
 """
+
+import numpy as np
+
+from confounds import Residualize
+from scipy import stats
+
+
+def partial_correlation(X, y=None):
+    """
+    Calculates the partial correlation
```

Comment:
elaborate on what exact variables are used for computing the metric .. e.g. "computes partial correlation between ... and ... ", and say something about when it can be used e.g. top 2 applications